### PR TITLE
Autocomplete components on mobile

### PR DESCRIPTION
**Pull request by joel-jeremy**
Reuse desktop's autocomplete components in mobile.

### CODE REVIEW

File: packages/desktop-client/src/components/mobile/MobileAmountInput.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ class AmountInput extends PureComponent {
         autoCapitalize="none"
         onChange={e => this.onChangeText(e.target.value)}
         onBlur={this.onBlur}
-        onKeyPress={this.onKeyPress}
+        onKeyUp={this.onKeyUp}
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/mobile/MobileAmountInput.js:154]: Updated since `onKeyPress` is now deprecated.

**Comment by MatissJanis** [comment]
The biggest benefit of the old autocomplete was that it opened up full-screen. Whereas the new one has a very small scroll window. For mobile devices the full-screen view IMO is much better.

WDYT @trevdor ?

Screenshot 2023-09-16 at 13 31 32
Screenshot 2023-09-16 at 13 31 29

**Comment by trevdor** [comment]
I agree. The mobile autocomplete should really be a “sheet” in the native sense of the word, not a simple drop down like on desktop. 

I suspect we’d better develop the mobile one further to be even more different from desktop rather than unify them. 

Apologies for not thinking of this sooner.

**Comment by trevdor** [comment]
> I suspect we’d better develop the mobile one further to be even more different from desktop rather than unify them.

I suppose this might be achievable in a unified component. But it could also prove too complicated trying to wrangle two very different designs in one.  

(and fwiw, in my opinion the existing mobile autocomplete is pretty much the goal—already emulated a sheet, though maybe with some larger touch targets eventually.

**Comment by joel-jeremy** [comment]
Sounds good! Just tried experimenting with it. Closing this PR. :)

**Pull request closed without merging**